CONTRIBUTING.md

```diff
@@ -27,25 +27,15 @@ To change the protocol (e.g. by adding a new field to some request parameters),
 the [Smithy](https://smithy.io/2.0/index.html) model.
 The model is located in the [spec/src/main/resources/META-INF/smithy/bsp](spec/src/main/resources/META-INF/smithy/bsp) folder.
 The model is then used to generate the protocol specification, the documentation and the support libraries.
-Support libraries can be generated by running `sbt generate`.
+Support libraries can be generated by running `bazel run //:generate`.
 
 ## Formatting
 
-To ensure the Scala sources are formatted correctly you can run:
-
-```sh
-sbt scalaFormat
-```
+Formatting will happen as a part of the generation process
 
 ## Releasing
 
-This repo is setup to use
-[sbt-ci-release](https://github.com/sbt/sbt-ci-release) for easy releasing. To
-publish a new release, you can go to the [releases
-page](https://github.com/build-server-protocol/build-server-protocol/releases)
-to tag a new release and auto-generate the Release Notes. The release notes are
-important as this is useful for downstream users to be able to click the
-"Release Notes" section of a Steward PR and be brought to this release.
+TODO
```

Comment:
Don't we want to figure this out before merging?

Reply:
Yup, Danil Tereshchenko from our team is working on it

Reply:
From the perspective of a (GitHub) user it works the same as before: you tag a release and the website is generated and deployed and artifacts are uploaded to maven.

The difference is that now we use https://github.com/JetBrains/bazel-sonatype instead of https://github.com/xerial/sbt-sonatype